federatedml/transfer_variable/definition/transfer_conf.json

```diff
@@ -875,6 +875,145 @@
    ]
   }
  },
+ "HomoFMTransferVariable": {
+  "guest_uuid": {
+   "src": "guest",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "host_uuid": {
+   "src": "host",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "uuid_conflict_flag": {
+   "src": "arbiter",
+   "dst": [
+    "guest",
+    "host"
+   ]
+  },
+  "dh_pubkey": {
+   "src": "arbiter",
+   "dst": [
+    "guest",
+    "host"
+   ]
+  },
+  "dh_ciphertext_host": {
+   "src": "host",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "dh_ciphertext_guest": {
+   "src": "guest",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "dh_ciphertext_bc": {
+   "src": "arbiter",
+   "dst": [
+    "guest",
+    "host"
+   ]
+  },
+  "paillier_pubkey": {
+   "src": "arbiter",
+   "dst": [
+    "host"
+   ]
+  },
+  "guest_model": {
+   "src": "guest",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "host_model": {
+   "src": "host",
+   "dst": [
+    "arbiter"
+   ]
+  },
+  "aggregated_model": {
+   "src": "arbiter",
+   "dst": [
+    "guest",
+    "host"
+   ]
+  },
+  "to_encrypt_model": {
```

Comment:
unused transfer variable should delete, like "pailler_pubkey"\"to_encrypt_model"\"re_encrypted_model"